pom.xml

```diff
@@ -178,7 +178,7 @@
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>data-import-processing-core</artifactId>
-      <version>3.6.0-SNAPSHOT</version>
+      <version>4.0.0-SNAPSHOT</version>
```

Comment:
Any reason we are not using a release version of this package?

Reply:
Yes, we need specific functional, from this package, that have not been released yet